api/tests/app/test_config.py

```diff
@@ -0,0 +1,219 @@
+import pytest
+
+from api.tests.utils.common_functions import get_config_file_json, app_config_file_multi_type_json_to_string_json
+# The line below is absolutely necessary. Fixtures are passed as arguments to test functions. That is why IDE could
+# not recognized them.
+from api.tests.utils.fixtures_tests import config_rollback, app_config
+
+
+def expected_response_update_report_info(config_sample_path):
+    app_config = get_config_file_json(config_sample_path)
+    expected_response = app_config_file_multi_type_json_to_string_json(app_config)
+
+    # It is weird that this change does not appear in response.json()
+    # Because if request was successfully, the PUT request will change the flag "has_been_configured"
+    expected_response['app']['has_been_configured'] = 'False'
+    return expected_response
+
+
+def body_without_app_field(client):
+    response_get_config = client.get('/config')
+    body = response_get_config.json()
+    del body["app"]
+    return body
+
+
+# pytest -v api/tests/app/test_config.py::TestClassGetProcessorInfo
+class TestClassGetProcessorInfo:
+    """Get Config File, GET /config/info"""
+
+    def test_get_processor_info(self, config_rollback):
+        """I dont get why has been configured is True"""
```

Comment:
Should this also be a `FIXME` comment?

Reply:
I think that it was fixed in the API refactor.

Reply:
So it's not an issue anymore?

Reply:
Yes. we can remove the comment from the test